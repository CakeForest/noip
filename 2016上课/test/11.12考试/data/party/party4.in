8 10
2 1
3 2
4 2
5 4
6 5
7 6
8 5
1 2 9
2 2 7
3 2 8
4 1 1
5 2 4
6 2 3
7 2 2
8 2 2
7 2 9
4 1 3
10
2
5
6
4
7
5
5
7
7
4
