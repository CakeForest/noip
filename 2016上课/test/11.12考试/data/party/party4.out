7
11
11
8
13
11
11
13
13
8
